src/views/profile.hbs

```diff
@@ -0,0 +1,106 @@
+<div id="modal-details" class="modal">
+  <div class="modal-background">
+    {{> profile_myDetails }}
+  </div>
+</div>
+
+
+
+<nav id="navigation">
```

Comment:
There is an `.hbs` file for the main navigation - `views > partials > header.hbs` so these link can be placed there.